src/GraphQL.SystemTextJson/ExecutionResultJsonConverter.cs

```diff
@@ -11,16 +11,16 @@ public override void Write(Utf8JsonWriter writer, ExecutionResult value, JsonSer
         {
             writer.WriteStartObject();
 
-            // Important: Don't pass the same options down when recursively calling Serialize.
+            // Important: Be careful with passing the same options down when recursively calling Serialize.
```

Comment:
@benmccallum In fact, there are no problems with passing down options if we serialize **another type** of object, no recursion.

Reply:
Good to see, that makes me feel way more comfortable that the options will be used in the lower parts.